gapic-generator-cloud/lib/gapic/generators/cloud_generator_parameters.rb

```diff
@@ -21,10 +21,12 @@ module Generators
     # Contains the cloud generator's parameters
     module CloudGeneratorParameters
       BOOL_PARAMETERS_ALIASES = {
-        "ruby-cloud-free-tier"         => ":gem.:free_tier",
-        "ruby-cloud-yard-strict"       => ":gem.:yard_strict",
-        "ruby-cloud-generic-endpoint"  => ":gem.:generic_endpoint",
-        "ruby-cloud-generate-metadata" => ":generate_metadata"
+        "ruby-cloud-free-tier"                    => ":gem.:free_tier",
+        "ruby-cloud-yard-strict"                  => ":gem.:yard_strict",
+        "ruby-cloud-generic-endpoint"             => ":gem.:generic_endpoint",
+        "ruby-cloud-generate-metadata"            => ":generate_metadata",
+        "ruby-cloud-generate-standalone-snippets" => ":generate_standalone_snippets",
```

Comment:
I try to add all new parameters into the `shared/test_resources/api_test_resources.rb` and the corresponding test cases because it then becomes easy to look at the whole tree of all the parameters filled.
Don't know if you want to do it or not, I can definitely do it myself later.

Reply:
I see. Let's do that in a separate PR. I think it could get kinda large.

Reply:
Opened #630 to remind us of this.